script.js

```diff
@@ -1,12 +1,126 @@
 //You can edit ALL of the code here
+
```

Comment:
Please review this file @PeteLindsell

Reply:
Hi Pete, I have made all the changes as requested. There is just one comment I need clarification on please. Thanks much.